src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -38,6 +45,78 @@ public static void ProcessLogFile(string filePath, Solution solution, string too
             if (toolFormat.MatchesToolFormat(ToolFormat.None))
             {
                 logText = File.ReadAllText(filePath);
+
+                if (logText.Contains(@"""version"": ""1.0.0"""))
```

Comment:
You need a regex because you also have to match `"version"   :   \t\n  "1.0.0"`.